src/main.py

```diff
@@ -45,12 +45,16 @@ def get_success_rate():
     training_lists = pickle_io.load_pickles("../data/training.dat")
     test_lists = pickle_io.load_pickles("../data/test.dat")
     print("Successfully loaded images from pickle files")
-    
+
     # Get reduces training and test images as tuple - reduced_images[0] is train_list, [1] is test_list without digits
-    reduced_images = pca.reduce_dimensions([csv_image.image for csv_image in training_lists], [csv_image.image for csv_image in test_lists])
+    reduced_images = pca.reduce_dimensions([csv_image.image for csv_image in training_lists], [csv_image.image for csv_image in test_lists], 784)
     print("PCA finished successfully")
-
+    for i in [10, 20, 50, 100, 200, 400, 784]:
+        pca.increase_dimensions(reduced_images[2], [red[:i] for red in reduced_images[1]], i)
```

Comment:
draw the pictures next to each other and display the according value for n

Reply:
done